lib/hashgraph.cc

```diff
@@ -276,29 +276,22 @@ void Hashgraph::consume_sequence_and_tag(const std::string& seq,
 //
 
 // TODO? Inline in header.
-void
-Hashgraph::
-consume_fasta_and_tag(
-    std:: string const  &filename,
-    unsigned int	      &total_reads, unsigned long long	&n_consumed
+template<typename SeqIO>
```

Comment:
Don't think so. If we want to back a decision with "for speed" we should measure it, otherwise we can just say "liked this better" :) Personal preference is `consume(fname, reader)` but that is mostly taste, not any objective reason.